src/indexables-page.php

```diff
@@ -0,0 +1,124 @@
+		$this->option = $option;
+	}
+
+	/**
+	 * Adds the required hooks for this class.
+	 */
+	public function add_hooks() {
+		\add_filter( 'wpseo_posts_threshold', [ $this, 'indexables_posts_threshold' ], 10, 1 );
+		\add_filter( 'wpseo_analyzed_posts_threshold', [ $this, 'indexables_analyzed_posts_threshold' ], 10, 1 );
+
+		\add_action( 'admin_post_yoast_seo_test_indexables_page', [ $this, 'handle_submit' ] );
+	}
+
+	/**
+	 * Filter minimum post threshold.
+	 *
+	 * @param int $number The current post threshold.
+	 *
+	 * @return int The current post threshold.
+	 */
+	public function indexables_posts_threshold( $number ) {
+		if ( $this->option->get( 'indexables_posts_threshold' ) > 0 ) {
+			return $this->option->get( 'indexables_posts_threshold' );
+		}
+
+		return $number;
+	}
+
+	/**
+	 * Filter minimum analyzed post threshold.
+	 *
+	 * @param int $number The current analyzed post threshold.
+	 *
+	 * @return int The current analyzed post threshold.
+	 */
+	public function indexables_analyzed_posts_threshold( $number ) {
+		if ( $this->option->get( 'indexables_analyzed_posts_threshold' ) > 0 ) {
+			return $this->option->get( 'indexables_analyzed_posts_threshold' );
+		}
+
+		return $number;
+	}
+
+	/**
+	 * Retrieves the controls.
+	 *
+	 * @return string The HTML to use to render the controls.
+	 */
+	public function get_controls() {
+		// phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedHooknameFound -- Using WPSEO hook.
+		$placeholder_thresholds = \apply_filters( 'wpseo_posts_threshold', Indexables_Page_Helper::POSTS_THRESHOLD );
+		// phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedHooknameFound -- Using WPSEO hook.
+		$placeholder_analyzed_thresholds = \apply_filters( 'wpseo_analyzed_posts_threshold', Indexables_Page_Helper::ANALYSED_POSTS_THRESHOLD );
+
+		$value = '';
+		if ( $this->option->get( 'indexables_posts_threshold' ) > 0 ) {
+			$value = $this->option->get( 'indexables_posts_threshold' );
+		}
+
+		$output  = '<label for="indexables_posts_threshold">' . \esc_html__( 'Minimum threshold for the amount of posts needed:', 'yoast-test-helper' ) . '</label>';
+		$output .= '<input type="number" size="5" value="' . $value . '" placeholder="' . $placeholder_thresholds . '" name="indexables_posts_threshold" id="indexables_posts_threshold"/><br/>';
+
+		$value = '';
+		if ( $this->option->get( 'indexables_analyzed_posts_threshold' ) > 0 ) {
+			$value = $this->option->get( 'indexables_analyzed_posts_threshold' );
+		}
+
+		$output .= '<label for="indexables_analyzed_posts_threshold">' . \esc_html__( 'The minimum threshold for the amount of analyzed posts:', 'yoast-test-helper' ) . '</label>';
+		$output .= '<input type="number" step=".01" size="5" min=0 max=100 value="' . $value . '" placeholder="' . $placeholder_analyzed_thresholds . '" name="indexables_analyzed_posts_threshold" id="indexables_analyzed_posts_threshold"/><span>%</span><br/>';
```

Comment:
As it is right now, it shows an incorrect value for the percentage, as I think it should be multiplied by 100.

For example, if no options are saved, it should show `50`% instead of `0.5`%